changelog/install-dmd-dc.dd

```diff
@@ -0,0 +1,27 @@
+`~/dlang/install.sh` now supports `--dmd` and `--dc`
+
+`--dmd` and `--dc` has been added as a new option to the install script.
+These flags work analogous to the existing `--activate` flag and
+print information about the installed compiler binary:
+
+$(UL
+    $(LI `--dmd` returns the path of a specific compiler executable (DMD-like interface))
+    $(LI `--dc` returns the path of a specific compiler executable (custom compiler interface))
+)
+
+$(H4 Examples)
+
+$(CONSOLE curl https://dlang.org/install.sh | bash -s --dmd
+/home/seb/dlang/dmd-2.078.2/linux/bin64/dmd
```

Comment:
What would `--dc` give you if the compiler is `dmd`?

Reply:
The same (see the tests). The changelog isn't intended to be a full documentation, but to raise awareness of new features or improvements. dlang.org/install.html will come with more extensive documentation.

Reply:
> What would --dc give you if the compiler is dmd?

I added an example + it's not the documentation, just a changelog.